pkg/cli/admin/prune/imageprune/prune_test.go

```diff
@@ -77,6 +79,9 @@ func TestImagePruning(t *testing.T) {
 		deployments                          kappsv1.DeploymentList
 		dcs                                  appsv1.DeploymentConfigList
 		rss                                  kappsv1.ReplicaSetList
+		sss                                  kappsv1.StatefulSetList
```

Comment:
upstream took `sset` as a short alias for StatefulSet, just fyi

Reply:
Adjusted.